specs/android/send_firebase_push_notification.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: Send a firebase push notification to local Android emulator
```

Comment:
How about `Send a Firebase push notification to a local Android emulator`?

Reply:
Very nice! Changed it in 9b44064 😄 Thanks!